packages/sd-jwt-vc/src/test/index.spec.ts

```diff
@@ -53,3 +76,108 @@ describe('App', () => {
     expect(encodedSdjwt).rejects.toThrowError();
   });
 });
+
+describe('Revocation', () => {
+  const { signer, verifier } = createSignerVerifier();
+  const sdjwt = new SDJwtVcInstance({
+    signer,
+    signAlg: 'EdDSA',
+    verifier,
+    hasher: digest,
+    hashAlg: 'SHA-256',
+    saltGenerator: generateSalt,
+    statusListFetcher(uri: string) {
```

Comment:
Can't we model this a bit more like we do with SD-JWT, so you provide a signer and verification callback for example. To me it's a bit hidden that the statuListFetcher must aslo verify the status list. If we move this to the sd-jwt repo we could reuse verification callback and there's the following steps:
- call fetcher (provided by user)
- decode jwt, validate structure 
- verify signature (provided by user)


Also for the fetcher in the future it might be nice to provide a default implementation based on fetch. Starting from node 18.something all environments (Node, Browser, React Native) now implement a global fetch we could use?

Reply:
A default fetcher is provided as a fallback, the fetch function. It is supported by node v18 and web browsers and since this lib is >=18 this should be fine